.github/workflows/integration-package-tests.yaml

```diff
@@ -4,13 +4,13 @@ on:
   pull_request:
     paths:
       - .github/workflows/integration-package-tests.yaml
-      - "src/**/*"
+      - "src/integrations/*/**.py"
```

Comment:
@chrisguidry hoping this is what you meant by changing it back! I also specified this to be `.py` files

Reply:
Oh sorry, no, this should be `src/**/*.py` or else it can't ever trigger when we add the `all-integration-tests` label

Reply:
gotcha gotcha!

Reply:
applied in [4cdab8e](https://github.com/PrefectHQ/prefect/pull/13714/commits/4cdab8e47d8b7f1d61697977cdbccea0033093a7)

Reply:
nope, butterfingers. applied in [bf408d0](https://github.com/PrefectHQ/prefect/pull/13714/commits/bf408d000c1213db9da59ff11f4ae83878eeecd5)